### ISSUE TITLE
Cannot grant camera permission in task module - shield icon missing

### ISSUE THREAD

**Issue by xelossan** [MicrosoftDocs/msteams-docs#8970]
We have a Teams app that uses media permissions to access device camera. Everything was working fine and users were able to grant required permissions for the app using the shield icon in top right corner of the task module popup (next to the x button).

Now, this shield button doesn't show anymore, so users are unable to grant the required permissions. Has something changed in latest Teams version and granting permissions is no longer supported in task modules? The icon is still showing correctly in personal and tab app, but we need it in task modules as well, as we trigger them from chat messages.

**Comment by Nivedipa-MSFT** [comment]
@xelossan - Thanks for reporting your issue. We will test this and update you soon.

**Comment by Jocelynncanzoneri** [comment]
This is Jocelynn the art and owner writer of this phone.  I have had difficulty's with doing anything on my phone.  Can you see what is interfering?

**Comment by xelossan** [comment]
@Nivedipa-MSFT we're using the latest version of both Teams SDK (@microsoft/teams-js 2.13.0) and Bot Builder SDK (Microsoft.Bot.Builder 4.20.0).

I cannot provide any code snippet as this is a part of much larger app, but I can give more details.

We have app that uses media permissions, defined as follows in app manifest:
`"devicePermissions": [
      "media"
    ],`

In our app, we open dialogs (previously called task modules) both from client side code, like that:
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/1189969/be738036-4980-4996-a3a4-325eccb5a961)
and from chat messages sent from our bot. Here, after users click the button in the message, we return standard TaskModuleResponse.

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/1189969/5171e34b-5e87-4e82-9281-776ee577a40b)

In both cases, everything worked fine previously - our app was launched in popup window, and user could allow access to the camera using the shield button.

Now, after one of Teams updates, but not sure which one, when popup is launched from chat message, the shield button is no longer displayed (it still works as expected when launched via Teams SDK).

**Comment by Nivedipa-MSFT** [comment]
@xelossan - Thanks for providing details. We will test this and update you soon.

**Comment by xelossan** [comment]
Yes, it was previously displaying shield icon when task module was opened from chat. It was the scenario that we were using in production, and it completely stopped working as users can no longer grant camera permissions.

**Comment by Nivedipa-MSFT** [comment]
@xelossan - We are checking this with engineering team, we will let you know once we get any updates on it.

**Comment by xelossan** [comment]
@Nivedipa-MSFT do you have any updates?

**Comment by Nivedipa-MSFT** [comment]
@xelossan - We have raised a bug for the same. Engineering team is working on it. We will let you know once we get any updates on it.

**Comment by Nivedipa-MSFT** [comment]
@xelossan - Engineering team has considered this as feature request. To achieve your requirements, we recommend you give your feedback in [Teams Feedback Portal](
https://feedbackportal.microsoft.com/feedback/forum/ad198462-1c1c-ec11-b6e7-0022481f8472)
